tendrl/gluster_integration/objects/volume_alert_counters/__init__.py

```diff
@@ -0,0 +1,22 @@
+from tendrl.commons import objects
+
+
+class VolumeAlertCounters(objects.BaseObject):
+    def __init__(
+        self,
+        warn_count=0,
+        integration_id='',
+        volume_id='',
+        *args,
+        **kwargs
+    ):
+        super(VolumeAlertCounters, self).__init__(*args, **kwargs)
+        self.warning_count = warn_count
+        self.integration_id = integration_id
+        self.volume_id = volume_id
+        self.value = '/clusters/{0}/Volumes/{1}/alert_counters'
+
+    def render(self):
+        self.value = self.value.format(
+            self.integration_id, self.volume_id)
```

Comment:
self.integration_id or NS.tendrl_context.integration_id

Reply:
i will change this